components/Statistics/__tests__/IconStat-test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render } from "@/test-utils/test-utils";
+import IconStat from "@/components/Statistics/IconStat";
+import { foofLightTheme, foofTheme } from "@/constants/custom-theme";
+
+describe("TourStats", () => {
+  it("should render the correct icon", async () => {
+    const view = render(<IconStat icon="home">1001km</IconStat>);
+    expect(view).toMatchSnapshot();
+  });
+  it("should render the icon and font in correct size", () => {
+    const view = render(
+      <IconStat icon="home" iconSize={30} fontSize={21}>
+        1001km
+      </IconStat>,
+    );
+    expect(view).toMatchSnapshot();
+  });
+  describe("should render the correct icon color depending on the given status", () => {
+    it("primary", () => {
```

Comment:
Hier vielleicht lieber das Keyword `test` statt `it` nehemen da es sich beser liest. Die machen aber exakt das gleiche